README.md

```diff
@@ -38,17 +38,20 @@ To setup the project locally read these wiki pages and follow the instructions:
 
 The project runs on Python 3.
 
-1. Create a virtual environment:
+1. Install latest version of virtualenv:
+`pip install -U virtualenv`
```

Comment:
Can you use ```

So that it looks like

```
pip install -U virtualenv
```

Reply:
Formatting is fine @SanketDG. It follows the same style as steps no. 2, 3,....

Reply:
Oh apologies, I meant, please use code blocks in all places for commands.

Reply:
This is not related to the pip issue. @SanketDG @PrashanthPuneriya  should I open another pull request for this.

Reply:
Please make the changes here since changes are already there in README file

Reply:
@SanketDG changes done. Please review again